lib/logger.js

```diff
@@ -268,11 +275,20 @@ Logger.prototype._log = function(info, level, levelPath, logger) {
 };
 
 Logger.prototype.log = function(level, info) {
+    var simpleLevel;
     if (!level || !info) {
         return;
     }
 
-    var simpleLevel = this._getSimpleLogLevel(level);
+    if (Logger.logTrace) {
+        simpleLevel = extractSimpleLevel(level);
+        if (simpleLevel) {
+            this._log(info, simpleLevel, level, this._traceLogger);
+            return;
```

Comment:
Should be out of the `if` block, since if `extractSimpleLevel` can't get the level, `_getSimpleLogLevel` won't be able to do so either.